src/sentry/api/endpoints/organization_stats_v2.py

```diff
@@ -0,0 +1,239 @@
+from rest_framework.response import Response
+from sentry.api.bases.organization import OrganizationEndpoint
+from sentry.api.utils import get_date_range_rollup_from_params
+from sentry.api.bases.project import ProjectEndpoint
+from sentry.models import Project, Team
+
+from sentry.snuba import outcomes
+from sentry_relay import DataCategory
+from sentry.utils.outcomes import Outcome
+
+from sentry.utils.snuba import (
+    naiveify_datetime,
+    to_naive_timestamp,
+)
+
+
+CATEGORY_NAME_MAP = {
+    DataCategory.ERROR: "statsErrors",
+    DataCategory.TRANSACTION: "statsTransactions",
+    DataCategory.ATTACHMENT: "statsAttachments",
+}
+
+
+class OrganizationStatsEndpointV2(OrganizationEndpoint):
+    def get(self, request, organization):
+        start, end, rollup = get_date_range_rollup_from_params(request.GET, "1h", round_range=True)
+        result = outcomes.query(
+            start=start,
+            end=end,
+            rollup=rollup,
+            groupby=["category", "time", "outcome", "reason"],
+            aggregations=[["sum", "times_seen", "times_seen"], ["sum", "quantity", "quantity"]],
+            filter_keys={"org_id": [organization.id]},
```

Comment:
Should this also filter by the projects you're allowed to see, but not expose that information in the breakdowns?